python/doc/user_manual/functions.rst

```diff
@@ -33,18 +33,28 @@ Algebra of functions
     LinearCombinationFunction
     ParametricFunction
     ProductFunction
-
-Field functions
-===============
+    FieldToFieldConnection
+    FieldToPointConnection
+    PointToFieldConnection
+    PointToPointConnection
```

Comment:
These Connection classes have no docstring, only copy paste of FieldToPointFunction.